### PR TITLE
Update set-up-multi-factor-authentication.md

### PR DESCRIPTION
**Pull request by msbemba**
Changed settings for MFA enabling in O365 as earlier it was more about conditional policies.

### CODE REVIEW

**Comment by msbemba** [comment]
Issue# https://github.com/MicrosoftDocs/OfficeDocs-O365ITPro/issues/874

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 1]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
 
 ## Set up multi-factor authentication
 
-1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Setup**.
+1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select select **Settings** \> **Services & add-ins**
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Settings** \> **Services & add-ins**
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 2]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
...
 
-2. Next to **Sign-in and security**, under **Make sign-in more secure**, select **View**.
+2. On the **Services & add-ins** page, select **‎Azure‎ multi-factor authentication**.
 
-3. On the **Make sign-in more secure** page, select **Get started**.
+3. Select the **User** Or Click On **Bullk Update** If you would like to enable it for more than one user.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
3. Select the user or click on **Bullk Update** if you would like to enable it for more than one user.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 3]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
...
 
-4. Select the **Require multi-factor authentication** and **Require users to register for multi-factor authentication and block access if risk is detected** check boxes.
+4. Click on **Enable** under **Quick Steps**  and Click on **Enable Multi-Factor Auth**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
4. Under **Quick Steps**, select **Enable** \> **Enable Multi-Factor Auth**.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 4]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
...
 ## Manage MFA settings 
 
-1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Setup**.
-
-2. Next to **Sign-in and security**, under **Make sign-in more secure**, select **View**.
-
-3. Under **Make sign-in more secure**, select **Manage**. 
-
-4. The **Azure portal Conditional Access - Policies** page will appear. To turn multi-factor authentication on or off:
+1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select select **Settings** \> **Services & add-ins**
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
1. In the [admin center](https://go.microsoft.com/fwlink/p/?linkid=834822), select **Settings** \> **Services & add-ins**
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 5]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
...
 
-    1. Select **Baseline policy: End user protection (Preview)**, and turn the **Enable** toggle on or off.
+2. On the **Services & add-ins** page, select **‎Azure‎ multi-factor authentication**.
 
-    2. Select **Baseline policy: Require MFA for admins (Preview)**, and turn the **Enable** toggle on or off.
+3. Select the **User** Or Click On **Bullk Update** If you would like to enable it for more than one user.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
3. Select the user or click on **Bullk Update** if you would like to enable it for more than one user.
```

File: Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md [hunk 6]
```diff
@@ -52,36 +52,26 @@ In SharePoint online, by default, modern authentication is enabled.
...
 
-    > [!NOTE]
-    > To exclude users from a policy, select **specific users excluded** > **Select excluded users**, select the users from the list, and then choose **Select**.
+4. Click on **Manage User Settings** under **Quick Steps**.
```
Comment (JohanFreelancer9) [inline Office365-Admin/security-and-compliance/set-up-multi-factor-authentication.md]:
```suggestion
4. Under **Quick Steps**, select **Manage User Settings**.
```

**Comment by mypil** [comment]
@msbemba  - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

**Comment by msbemba** [comment]
> @msbemba - please merge the commit on the copy/edit review done by JohanFreelancer9 or make the necessary changes on the PR following the suggestions made. Thank you.

It is complete now. Thank you.

**Comment by mypil** [comment]
@Sirkkuw - PR has been copy edited and is ready for final review. This is based on issue #874

Thank you.

cc: @kenwith

**Comment by Sirkkuw** [comment]
I am not ready to merge this yet. Need to double check.

**Comment by Sirkkuw** [comment]
Actually - I am wanting to reject this - the new Setup experience is like it is in the topic. This would revert it to the old way, It is possible that some tenants see the old way still.

**Comment by ragupt** [comment]
Hi @Sirkkuw  - Good morning / Evening - I tried purchasing a new tenant ( multiples ) and I don't see the behavior mentioned in this article. neither I saw any documentation for that behavior in office 365 roadmap

**Comment by Sirkkuw** [comment]
@Ragupt - did you purchase a tenant. and I am wondering if this is a locale issue? Baseline policies are being deprecated and being replaced by Security defaults. The topic now reflects that. However, if you are seeing a different UI, please take a screenshot and I will go ahead and ask the product team as to what is going on.

**Pull request closed without merging**